[BUG] Use of `# ignore: type` leading to mypy error in CI

### Description

CI is failing due to mypy throwing an error:

```
across/tools/core/schemas/bandpass.py:88: error: Unused "type: ignore" comment  [unused-ignore]
across/tools/core/schemas/bandpass.py:90: error: Unused "type: ignore" comment  [unused-ignore]
Found 2 errors in 1 file (checked 39 source files)
across/tools/core/schemas/bandpass.py:88: error: Unused "type: ignore" comment  [unused-ignore]
across/tools/core/schemas/bandpass.py:90: error: Unused "type: ignore" comment  [unused-ignore]
Found 2 errors in 1 file (checked 38 source files)
```

The `type: ignore` used here can be removed by explicitly typing values that are converted from `float` to `float | Any | None` due to `astropy` not being type-aware. 

### Steps to Reproduce

1. Look here: https://github.com/ACROSS-Team/across-tools/actions/runs/15565411556/job/43828155010?pr=33

### Expected Behavior

Mypy does not throw this error.

--- a/across/tools/core/schemas/bandpass.py
+++ b/across/tools/core/schemas/bandpass.py
@@ -75,19 +75,19 @@ class WavelengthBandpass(BaseBandpass):
         if not all([self.central_wavelength > 0, self.bandwidth > 0]):
             raise BandwidthValueError("Central wavelength and bandwidth must be positive.")
 
-        self.bandwidth = (
+        self.bandwidth = float(
             (self.bandwidth * u.Unit(self.unit.value)).to(u.Unit(WavelengthUnit.ANGSTROM.value)).value
         )
 
-        self.central_wavelength = (
+        self.central_wavelength = float(
             (self.central_wavelength * u.Unit(self.unit.value))
             .to(u.Unit(WavelengthUnit.ANGSTROM.value))
             .value
         )
 
-        self.min = self.central_wavelength - (self.bandwidth)  # type: ignore
+        self.min = self.central_wavelength - self.bandwidth
 
-        self.max = self.central_wavelength + (self.bandwidth)  # type: ignore
+        self.max = self.central_wavelength + self.bandwidth
 
         self.unit = WavelengthUnit.ANGSTROM
 

